packages/perseus-editor/src/components/locked-figure-settings-actions.tsx

```diff
@@ -5,29 +5,72 @@
  */
 import Button from "@khanacademy/wonder-blocks-button";
 import {View} from "@khanacademy/wonder-blocks-core";
+import IconButton from "@khanacademy/wonder-blocks-icon-button";
+import {Spring} from "@khanacademy/wonder-blocks-layout";
 import {spacing} from "@khanacademy/wonder-blocks-tokens";
+import caretDoubleDownIcon from "@phosphor-icons/core/bold/caret-double-down-bold.svg";
+import caretDoubleUpIcon from "@phosphor-icons/core/bold/caret-double-up-bold.svg";
+import caretDownIcon from "@phosphor-icons/core/bold/caret-down-bold.svg";
+import caretUpIcon from "@phosphor-icons/core/bold/caret-up-bold.svg";
 import trashIcon from "@phosphor-icons/core/bold/trash-bold.svg";
 import {StyleSheet} from "aphrodite";
 import * as React from "react";
 
+import type {LockedFigureType} from "@khanacademy/perseus";
+
+export type LockedFigureSettingsMovementType = "top" | "up" | "down" | "bottom";
+
 type Props = {
+    figureType: LockedFigureType;
+    onMove: (movement: LockedFigureSettingsMovementType) => void;
     onRemove: () => void;
-    figureAriaLabel: string;
 };
 
 const LockedFigureSettingsActions = (props: Props) => {
-    const {onRemove, figureAriaLabel} = props;
+    const {figureType, onMove, onRemove} = props;
+
     return (
         <View style={styles.container}>
             <Button
                 startIcon={trashIcon}
-                aria-label={`Delete ${figureAriaLabel}`}
+                aria-label={`Delete locked ${figureType}`}
                 onClick={onRemove}
                 kind="tertiary"
                 style={styles.deleteButton}
             >
                 Delete
             </Button>
+
+            <Spring />
+
+            <IconButton
+                icon={caretDoubleUpIcon}
+                size="small"
+                aria-label={`Move locked ${figureType} to the top`}
+                onClick={() => onMove("top")}
+                style={styles.iconButton}
+            />
+            <IconButton
+                icon={caretUpIcon}
+                size="small"
+                aria-label={`Move locked ${figureType} up`}
+                onClick={() => onMove("up")}
+                style={styles.iconButton}
+            />
+            <IconButton
+                icon={caretDownIcon}
+                size="small"
+                aria-label={`Move locked ${figureType} down`}
+                onClick={() => onMove("down")}
+                style={styles.iconButton}
+            />
+            <IconButton
+                icon={caretDoubleDownIcon}
+                size="small"
+                aria-label={`Move locked ${figureType} to the bottom`}
```

Comment:
Minor: in graphics programs, the usual labels for these concepts are "bring to front" and "send to back". E.g. here's how Google Drawings does it:

<img width="617" alt="Screen Shot 2024-06-20 at 11 05 40 AM" src="https://github.com/Khan/perseus/assets/693920/931873a1-67db-419b-a4fd-20d39f99d6ea">

"front"/"forward"/"back"/"backward" might be more recognizable than "top"/"up"/"down"/"bottom".